www/js/controllers/company_home_controller.js

```diff
@@ -4,8 +4,8 @@ app.controller('CompanyHomeCtrl', function($scope, $state, $ionicLoading, Barcod
 
   if ($scope.currentUser.firstLogin) {
     NotificationService.showAlert({
-      title: "This device is now associated with " + $scope.currentUser.company + ".",
-      template: "To set your preferences, please see the main menu."
+      scope: $scope,
+      templateUrl: "templates/company-association.html"
```

Comment:
I don't think it's necessary to use a custom template here given that it's just title and body. Let's just add a method to the notificationservice that takes an optional title as the second argument.